pkg/sdk/poc/generator/operation.go

```diff
@@ -161,7 +162,17 @@ func (i *Interface) ShowOperation(doc string, dbRepresentation *dbStruct, resour
 }
 
 func (i *Interface) ShowByIdOperation() *Interface {
-	return i.newNoSqlOperation(string(OperationKindShowByID))
+	op := newNoSqlOperation(string(OperationKindShowByID))
+	i.Operations = append(i.Operations, op)
+	return i
+}
+
+func (i *Interface) ShowByIdOperationWithFiltering(filter ShowByIDFilteringKind, filtering ...ShowByIDFilteringKind) *Interface {
```

Comment:
New Operation forcing users to use filtering options (as discussed with @sfc-gh-asawicki)